complete/src/main/java/com/example/restservice/service/QueueService.java

```diff
@@ -24,10 +25,30 @@
   @Autowired private UserRepository userRepository;
   @Autowired private UserService userService; // TODO remove
 
+  private Random randomNumber = new Random();
```

Comment:
One thing I find, not in this PR, but through out the code is, that we don't use final. I think we need to, later, do a complete run on the code and make most of the members final.